### PR TITLE
move computed web3Account to useWeb3 composable

### PR DESCRIPTION
**Pull request by mktcode**
Reduces code duplication a bit.

`const web3Account = computed(() => web3.value.account);` was repeated in several components/composables. Moved it to the `useWeb3` composable.

### CODE REVIEW

File: src/components/Plugin/CommentBox/Comment.vue [hunk 1]
```diff
@@ -18,14 +18,13 @@ const props = defineProps({
   mainThread: String,
   space: Object
 });
-const { web3 } = useWeb3();
+const { web3Account } = useWeb3();
```
Comment (samuveth) [inline src/components/Plugin/CommentBox/Comment.vue:21]: We can remove the computed import

File: src/components/Plugin/CommentBox/CustomBlock.vue [hunk 2]
```diff
@@ -16,9 +16,8 @@ const props = defineProps({
 });
 const { notify } = useNotifications();
 const auth = getInstance();
-const { web3 } = useWeb3();
+const { web3Account } = useWeb3();
```
Comment (samuveth) [inline src/components/Plugin/CommentBox/CustomBlock.vue:19]: We can remove the computed inport

File: src/components/Plugin/CommentBox/Comment.vue (hunk 1 above)
Comment (samuveth) [reply src/components/Plugin/CommentBox/Comment.vue:21]: This applies to some other files too

**Review by samuveth — APPROVED** [review]
Looks good!

File: src/components/Plugin/CommentBox/Comment.vue (hunk 1 above)
Comment (mktcode) [reply src/components/Plugin/CommentBox/Comment.vue:21]: Yep. I think we should really add the linting ass a commit hook or so.

**Pull request merged**